src/url.ts

```diff
@@ -12,10 +14,11 @@ export function getBaseUrlFromIndentifier(
     // Remove anything that looks like a port temporarily since it confuses the URL parser
     url = new URL(identifier.replace(/:[0-9]+$/, ``));
   } catch (_e) {
+    const protocol = preferHtps ? "https" : 'http';
     // If it looks like a host, add a protocol, other let's assume it's a slug
     url = /\.|:/.test(identifier)
-      ? new URL(`https://${identifier}`)
-      : new URL(`https://${identifier}.${DEFAULT_ZIPPER_DOT_RUN_HOST}`);
+      ? new URL(`${protocol}://${identifier}`)
+      : new URL(`${protocol}://${identifier}.${zipperRunUrl}`);
   }
 
   if (overrideHost) url.host = overrideHost;
```

Comment:
FWIW, overrideHost is supposed to be the way to do this. Perhaps it should be `overrideOrigin` to include a protocol too?

Reply:
The main issue is that `overrideHost` updates the identifier too

Reply:
So you need to specify the applet identifier twice
`initApplet('hello-world', { overrideHost: 'hello-world.localdev.me' })`

I'm aiming in this usage:
```ts
const localhostApplet = (name, options) => initApplet(name, { ...options, host: 'localdev.me' })
localhostApplet('hello-world').run(...)
```

Reply:
Should I continue with the custom `zipperRunUrl` approach?

Reply:
ah got it. In that case, I think the we should remove `overrideHost` and make `zipperRunUrl` include the protocol and the port, so its only one option. 

Curious, what happens if you do `initApplet('https://hello-world.localdev.me:3002')`, does that work?

Reply:
Now it works, previously the port was missing